app/controllers/AssignedCasesController.scala

```diff
@@ -54,8 +54,7 @@ class AssignedCasesController @Inject() (
     for {
       cases            <- casesService.getAssignedCases(NoPagination())
       queues           <- queuesService.getAll
-      caseCountByQueue <- casesService.countCasesByQueue(request.operator)
-    } yield Ok(views.html.assigned_cases(queues, cases.results, assigneeId, caseCountByQueue, startAtTabIndex))
+    } yield Ok(views.html.assigned_cases(queues, cases.results, assigneeId, Map.empty, startAtTabIndex))
```

Comment:
This controller is not use anywhere anymore but I can't delete it because removing all of the links to it is another (quite big) ticket